tests/tests_geomstats/test_visualization_spd.py

```diff
@@ -0,0 +1,66 @@
+"""Unit tests for visualization."""
+
+import matplotlib.pyplot as plt
+import tests.conftest
+from geomstats.geometry.spd_matrices import SPDMatrices
+import symmetric_positive_definite
+
+
+class TestVisualizationSPD(tests.conftest.TestCase):
+    """Test cases for symmetric_positive_definite.py."""
+
+    def setup_method(self):
+        """Set up parameters for the following tests."""
+        self.n_samples = 10
+        self.spd = SPDMatrices(n=2)
+        self.max_z = 1
+        self.curr_z = 0.3
+        self.hsv = False
+        plt.figure()
+
+    def test_plot(self):
+        """Test case for function plot."""
+        viz = symmetric_positive_definite.SymPosDefVisualization(self.max_z)
```

Comment:
instantiate the object in the set up method.
self.spd_viz = SymPosDefVisualization(self.max_z)
and then call it everywhere with self.spd_viz